mautic/mautic
# Modify the form label

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [x]
| BC breaks? (use the c.x branch)        | [x]
| Automated tests included?              | [x] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


The form label and tooltip were modified. 

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new campaign or use an existing one
3. Add an email to the campaign 
4. Select the "in relative time period" option
5. New form label here
![image](https://user-images.githubusercontent.com/96085911/168800975-661bba7f-61b8-4f4b-a2c2-8784ff522e25.png)

![image](https://user-images.githubusercontent.com/96085911/168808176-3cdbcb15-8ac5-48f0-8de7-d3eea5d4e578.png)






<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11166"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
